ticdc/manage-ticdc.md

```diff
@@ -103,9 +103,13 @@ Info: {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":
 - `--start-ts`: Specifies the starting TSO of the `changefeed`. From this TSO, the TiCDC cluster starts pulling data. The default value is the current time.
 - `--target-ts`: Specifies the ending TSO of the `changefeed`. To this TSO, the TiCDC cluster stops pulling data. The default value is empty, which means that TiCDC does not automatically stop pulling data.
 - `--sort-engine`: Specifies the sorting engine for the `changefeed`. Because TiDB and TiKV adopt distributed architectures, TiCDC must sort the data changes before writing them to the sink. This option supports `memory`/`unified`/`file`.
-    - `memory`: Sorts data changes in memory. It is recommended to use `memory` in a production environment.
-    - `unified`: An experimental feature introduced in v4.0.9. When `unified` is used, TiCDC prefers data sorting in memory. If the memory is insufficient, TiCDC automatically uses the disk to store the temporary data. It is **NOT** recommended to use it in a production environment unless `memory` cannot be used due to insufficient memory.
-    - `file`: Entirely uses the disk to store the temporary data. This feature is **deprecated**. It is not recommended to use it.
+
+    - `memory`: Sorts data changes in memory.
+    - `unified`: When `unified` is used, TiCDC prefers data sorting in memory. If the memory is insufficient, TiCDC automatically uses the disk to store the temporary data. This is the default value of `--sort-engine`.
+    - `file`: Entirely uses the disk to store the temporary data. This feature is **deprecated**. It is **NOT recommended** to use it in **any** situation.
```

Comment:
```suggestion
    - `file`: Entirely uses the disk to store the temporary data. This option is **deprecated**. It is **NOT recommended** to use it in **any** situation.
```

Reply:
`--sort-engine` is an option. `memory`,`unified`, and `file` are optional values for `--sort-engine`. On L105, it says "This option supports `memory`/`unified`/`file`". Will it be confusing if we also use the word "option" here?

Reply:
Yes, you're right. Ignore this comment.

btw. — if it's no recommended to use `file` in **any situation**, why document it at all?

Reply:
@liuzix PTAL.

Reply:
> Yes, you're right. Ignore this comment.
> 
> btw. — if it's not recommended to use `file` in **any situation**, why document it at all?

There were users who tried to use `file`. The documentation is designed to explicitly warn about this.